hls4ml/utils/example_models.py

```diff
@@ -6,17 +6,18 @@
 
 from .config import create_config
 
+ORGANIZATION = 'fastmachinelearning'
+BRANCH = 'master'
+
 
 def _load_data_config_avai(model_name):
     """
     Check data and configuration availability for each model from this file:
 
-    https://github.com/hls-fpga-machine-learning/example-models/blob/master/available_data_config.json
+    https://github.com/fastmachinelearning/example-models/blob/master/available_data_config.json
```

Comment:
Is ORGANIZATION just for githubusercontent? I was wondering why it's not used here.

Reply:
I didn't change it in docstrings to not confuse people. Otherwise I would have to explain what `ORGANIZATION` and `BRANCH` are :smile:. githubusercontent goes to actual, raw files, while github links are webpages about files